.cirrus.yml

```diff
@@ -74,6 +74,17 @@ skip_task_on_pr: &SKIP_TASK_ON_PR
   skip: >
     ($CIRRUS_PR != '' && $CIRRUS_PR_LABELS !=~ '.*fullci.*')
 
+
+benchmark_only_if_template: &BENCHMARK_ONLY_IF_TEMPLATE
+  # only_if condition for cron-triggered benchmarking tests.
+  # These currently do not run for release/.*
+  only_if: >
+    ( $CIRRUS_REPO_NAME == 'zeek' || $CIRRUS_REPO_NAME == 'zeek-security' ) &&
+    ( $CIRRUS_CRON == 'benchmark-nightly' ||
+      $CIRRUS_PR_LABELS =~ '.*fullci.*' ||
+      $CIRRUS_PR_LABELS =~ '.*benchmark.*' ||
+      $CIRRUS_BRANCH =~ 'topic/awelzel/ubuntu22-spicy-task' )
```

Comment:
To be removed during merge.

Reply:
I'd be curious what the ultimate condition would look like. Also curious whether it would make sense to trigger this even on PRs if something in `src/spicy` or `auxil/spicy` was touched (this would assume that analyzers do not change much).

Reply:
I was forth and back to add `changesInclude()`, but I'm now at the point where I think the nightly signal is reasonable.

If for a certain spicy-bump or Zeek side change you do want to include the tasks, can put `benchmark` into the PR labels.